admin/test/test_release.py

```diff
@@ -1977,9 +1977,11 @@ def setUp(self):
         # Making a recipe involves interacting with PyPI, this should be
         # a parameter, not a patch. See:
         # https://clusterhq.atlassian.net/browse/FLOC-1759
-        self.patch(release, 'make_recipe',
-            lambda version, sdist_url:
-                "Recipe for " + version + " at " + sdist_url)
+        self.patch(
+            release, 'make_recipe',
+            lambda version, sdist_url, requirements_path:
+            "Recipe for " + version + " at " + sdist_url
+        )
```

Comment:
Ah, I see! *This* is why we didn't catch the problem in tests. (No action required)